lib/ProductOpener/Display.pm

```diff
@@ -1304,7 +1312,42 @@ sub display_mission($)
 	exit();
 }
 
+sub get_cache_results($$){
+
+	my $key = shift;
+	my $request_ref = shift;
+	my $results;
+
+	$log->debug("MongoDB hashed query key", { key => $key }) if $log->is_debug();
+
+	# disable caching if ?nocache=1
+	# or if the user is logged in and nocache is different from 0
```

Comment:
@stephanegigandet is it still relevant to disable cache when user is logged in ?